tests/test_unit.py

```diff
@@ -283,7 +283,13 @@ def test_config_development_settings(self):
             self.assertEqual(conf.db_pool_timeout, 3)
 
 
-@patch("app.Config", **{"return_value.mgmt_url_path_prefix": "/"})
+@patch(
+    "app.Config",
+    **{
+        "return_value.mgmt_url_path_prefix": "/",
+        "return_value.db_uri": "postgresql://insights:insights@localhost/insights",
```

Comment:
Even if this is not used for any destructive operation, I still feel better if we don’t use the “production” database name.

```suggestion
        "return_value.db_uri": "postgresql://insights:insights@localhost/insights_test",
```

But! I don’t think it is the right thing to do. This tests should only verify that the configuration has been loaded correctly. Now, the _atexit_ handler causes SQLAlchemy to do its teardown, potentially touching the database. I’d rather prevent somehow the teardown from being triggered.

Patching _db.get_engine_ should do the trick:

```python
@patch("app.db.get_engine")
```

I was thinking about adding some method into [_RuntimeEnvironment_](https://github.com/RedHatInsights/insights-host-inventory/blob/8f5c54164bbaff92a305114776f3ac3c761c9b10/app/environment.py). But we want to prevent it only in this specific tests. API tests use the database and thus should even do a proper teardown.